### DISCUSSION TITLE
Simple mikeio.DataArray Calculation

### DISCUSSION THREAD

**Discussion by bakjee** [DHI/mikeio#400] [Q&A] [answered]
Hi all, trying to work this one out for a couple of days.
So I have a DFSU file from both TR and HD simulation, trying to plot a time series of exchange rate of seawater in the selected domain.

In my HD output, I have the total water depth which i multiply with the concentration output from TR. I need to sum all of the multiplied values in each time step and plot the time-series.

It sounds simple enough and I have done it with my old code last year but for some reason it just doesnt work any more and my python skill is too mediocre and rusty, cant get my head around it and fix the old code

Thanks

**Comment by ecomodeller** [comment, marked as answer]
I certainly think it should be simple. Could it be something like this that you are looking for?
![image](https://user-images.githubusercontent.com/614215/184096818-1c6f443a-f057-42cb-9bd4-3a4e3a2842dc.png)

**Reply by bakjee** [reply]
thank you so much Henrik!! this was indeed simple and very helpful.
I noticed that mikeio.DataArray isn't similar to numpy arrays. If i could convert mikeio.Dataarray into np.array or dataframe,
I think i will be able to play around with the data for analysis and data export purposes with my limited python skills
Any suggestion for this?

**Reply by ecomodeller** [reply]
You can always get the data in the format of a numpy from a mikeio.DataArray using the `to_numpy()` method.

E.g. 
```python
ds = mikeio.read("file.dfsu")
da = ds.Total_water_depth
z = da.to_numpy()
```
